Sources/PublicInterface/Session.swift

```diff
@@ -40,12 +40,14 @@ public struct Session: Codable {
         public let description: String?
         public let icons: [URL]
         public let url: URL
+        public let scheme: String?
 
-        public init(name: String, description: String?, icons: [URL], url: URL) {
+        public init(name: String, description: String?, icons: [URL], url: URL, scheme: String? ) {
```

Comment:
can we add the default value for the scheme parameter, so that this won't break existing code?

```
scheme: String? = nil)
```

Reply:
Good call!